x/bank/simulation/operations_test.go

```diff
@@ -123,25 +124,31 @@ func (suite *SimTestSuite) TestSimulateMsgMultiSend() {
 }
 
 func (suite *SimTestSuite) TestSimulateModuleAccountMsgSend() {
-	s := rand.NewSource(1)
+	moduleAccCount := 2
+	s := rand.NewSource(int64(moduleAccCount))
 	r := rand.New(s)
 	accounts := suite.getTestingAccounts(r, 1)
 
-	addr := suite.app.AccountKeeper.GetModuleAddress(distributiontypes.ModuleName)
-	//addr := suite.app.AccountKeeper.GetModuleAddress(types.ModuleName) // use multiple module accounts.
-
-	acc := suite.app.AccountKeeper.GetAccount(suite.ctx, addr) // types.AccountI
-	//sacc, ok := acc.(simtypes.Account)
-
-	mAcc := simtypes.Account{
-		Address: acc.GetAddress(),
-		PrivKey: nil,
-		ConsKey: nil,
-		PubKey: acc.GetPubKey(),
+	for i := 0; i < moduleAccCount; i++ {
+		var addr sdk.AccAddress
+
+		switch {
+		case r.Int()%2 == 0:
```

Comment:
Added for loop here for simplicity and pick ModuleName in GetModuleAddress at random.